templates/home/redmine/redmine/config/configuration.yml.j2

```diff
@@ -3,15 +3,17 @@ default:
   # Outgoing emails configuration
   email_delivery:
 {% if redmine_enable_smtp_email %}
-    delivery_method: smtp
+    delivery_method: :smtp
```

Comment:
"smtp" alone is not a valid option (https://www.redmine.org/projects/redmine/wiki/EmailConfiguration#delivery_method)